fvm/fvm_bench_test.go

```diff
@@ -366,6 +443,26 @@ func BenchmarkRuntimeTransaction(b *testing.B) {
 	b.Run("emit event", func(b *testing.B) {
 		benchTransaction(b, templateTx(100, `TestContract.emit()`))
 	})
+	b.Run("borrow array from storage", func(b *testing.B) {
```

Comment:
nit: could use explanation how / why we want to compare benchmarks of borrow vs copy